### ISSUE TITLE
Does not change the font in the context menu on 1.77.3

### ISSUE THREAD

**Issue by radqut** [blackmann/fonted#2]
```
Version: 1.77.3 (user setup)
Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710
Date: 2023-04-12T09:16:02.548Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: No
```
![Screenshot_1](https://user-images.githubusercontent.com/50011226/235484654-896463d5-d591-4126-a982-43a77260c6ea.png)

**Comment by blackmann** [comment]
The context menu is the system ui. I don't think the CSS can change that.

**Comment by radqut** [comment]
No, it's not the system ui. You can change it in the `vs\workbench\workbench.desktop.main.js`. Just need replace `:host-context(.OS) { font-family: "Segoe WPC", "Segoe UI", sans-serif; }` in the file.

**Comment by radqut** [comment]
@blackmann I fixed it [here](https://github.com/blackmann/fonted/pull/3).

**Comment by Bohatcom** [comment]
@MichailShcherbakov @blackmann  Sounds great! Does it also fix issue with some extensions not using the Fonted font, eg Git Graph does not use the Fonted font. (I'm using VSCode on Windows)
Can this be merged?

**Comment by everythingsucksass** [comment]
Hi @Bohatcom, I wrote an extension yesterday using ChatGPT - https://marketplace.visualstudio.com/items?itemName=everythingsucks.ui-font-changer

It's very janky, but it works.

**Comment by Bohatcom** [comment]
@everythingsucksass Brilliant!  This is exactly what I was looking for. It changes the font everywhere including extensions such as GitGraph! None of the other extensions I've used over the past few years (including Fonted) could do this. Great work!

**Comment by everythingsucksass** [comment]
@Bohatcom Thank you!

**Comment by realwyz** [comment]
Since it did't updated, I looked for another extension, but it could only modify the context menu, and the interface of other extensions, e.g. Git Graph. So I'm using both of them at the same time right now😂. I hope the author will come back to accept this pull request.
